base/steer/FairRootManager.h

```diff
@@ -343,6 +345,12 @@ class FairRootManager : public TObject
      */
     std::map<TString, TObject*> fMap;   //!
 
+    /**folder name variable*/
+    static std::string fFolderName;
+    /**tree name variable*/
+    static std::string fTreeName;
```

Comment:
Please append a `   //!`.
I learned, that this will remove the fields from root's serializer.

OR if they should be serialized, consider bumping the version.